README-CONAN.md

```diff
@@ -1,108 +1,193 @@
+conan](http://docs.conan.io/en/latest/installation.html)).
 
-## Main concepts
+To upgrade the version of conan:
 
-Even when the following instructions will allow you to use conan directly, we strongly recommend to take a look to their
-"Getting started" [page](http://docs.conan.io/en/latest/getting_started.html), to better understand the main concepts.
+```bash
+$ pip install conan --upgrade
+```
 
-In the root level of the **Exiv2** repository, there is a file named `conanfile.py` that defines the C/C++ dependencies
-of the project. One of the dependencies is **Expat**, and it is stated like this:
+<name id="1-2"></a>
+##### 1.2) Test conan installation
 
-```python
-self.requires('Expat/2.2.5@pix4d/stable')
+```bash
+$ conan --version
+Conan version 1.4.1
 ```
 
-The syntax stands for: `Library/version@user/channel`.
+<name id="1-3"></a>
+##### 1.3) Create a build directory<name id="1-3"></a>
 
-Conan will search for these dependencies in a list of remotes. In those servers there are recipes (the python code that
-explains how to build a library from its sources) and binary packages (different versions of the library for several
-combinations of configurations: Platform/Compiler/Settings). If there is already a packages for your configuration,
-conan will fetch them. Otherwise, it will compile the libraries for you by using the instructions provided in the
-recipe. To better illustrate this idea, here we show a list of packages that is returned by the command `conan search`
+Create a build directory and will run the conan commands:
 
 ```bash
-conan search -r conan-center zlib/1.2.11@conan/stable
-
-   # conan-center zlib/1.2.11@conan/stable (This is the recipe for which we were searching packages)
-
-   Package_ID: ff0b79bd9b0fc1d915200da2fe02ae326cef4b1b
-       [options]
-           shared: False
-       [settings]
-           arch: x86_64
-           build_type: Debug
-           compiler: gcc
-           compiler.exception: seh
-           compiler.threads: posix
-           compiler.version: 4.9
-           os: Windows
-       Outdated from recipe: False
-
-   Package_ID: ff5ec8938c949149b25b89d656bed297cd5d5aad
-       [options]
-           shared: False
-       [settings]
-           arch: x86
-           build_type: Debug
-           compiler: clang
-           compiler.version: 3.9
-           os: Linux
-       Outdated from recipe: False
-
-   Package_ID: ff82a1e70ba8430648a79986385b20a3648f8c19
-       [options]
-           shared: True
-       [settings]
-           arch: x86_64
-           build_type: Debug
-           compiler: Visual Studio
-           compiler.runtime: MDd
-           compiler.version: 15
-           os: Windows
-       Outdated from recipe: False
+$ mkdir build
+$ cd build
+$ conan profile list
 ```
 
-## Installation
+**IMPORTANT** _**Visual Studio Users**_ require the profile msvc2017Release64 in %HOMEPATH%/.conan/profile/msvc2017Release64
```

Comment:
[suggestion] I am not sure if it is good to have here this note. This is duplicated, since it appears already in the section 2.2. I would propose to avoid these details in the step by step section, since anyways, conan creates a default profile with similar options. 

Actually this step title is "1.3 Create a build directory" which should just be `mkdir build && cd build`. The additional command is irrelevant to the title.

Reply:
I expressed it this way because on cmd.exe the syntax is `mkdir build && cd build`.  I think it's rather important to run `$ conan profile list` to let the user know what profiles are available.

An alternative way is to provide different "step by step" instructions for Visual Studio Users.  I was trying to make conan look simple.  If I say "Visual Studio Users" need   msvc2017Release64, I feel it's essential to say what's to be in that file.

You are quite correct is saying that the name of the file isn't important, however I'm trying to make this easy for folks who have never used conan and puzzled that changing defaults is involved.  @D4N has pointed out the MS provide VMs for testing Edge.  I've downloaded that and have it working.  So, now I have a "clean" Windows 10 onto which I'll load Visual Studio 2017 and see what default conan creates "out of the box".